django/http/response.py

```diff
@@ -37,10 +100,7 @@ class HttpResponseBase:
     status_code = 200
 
     def __init__(self, content_type=None, status=None, reason=None, charset=None):
-        # _headers is a mapping of the lowercase name to the original case of
-        # the header (required for working with legacy systems) and the header
-        # value. Both the name of the header and its value are ASCII strings.
-        self._headers = {}
+        self.headers = ResponseHeaders({})
```

Comment:
Is the new `__init__()` kwarg documented? (Can't see it.) 

What's the expected usage? Am I meant to pass a `ResponseHeaders` instance? (I **want** to pass a dictionary...)

Reply:
Maybe I'm too asleep to understand you, but there's no new kwarg. I do _plan_ one, but I'll write another ticket for that as Adam suggested on the list. And yes, the interface will be as you suggest, pass in a plain dict with your headers.

Reply:
Oh, you're right! Code blindness. Sorry for the noise. Doh.